src/test/java/hudson/plugins/junitattachments/GetTestDataMethodObjectTest.java

```diff
@@ -0,0 +1,12 @@
+package hudson.plugins.junitattachments;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class GetTestDataMethodObjectTest {
```

Comment:
What is the point of this test class?

Reply:
it is empty, yes, I can delete. But might be good to have for future..